triage/admin_billing_summary/cli.py

```
@@ -0,0 +1,211 @@
+    y, m = month_key.split("-")
+    from calendar import month_name
+    return f"{month_name[int(m)]}_{y}_Admin_Billing_Summary_MyPreferredFormat.xlsx"
+
+
+def _read_prior_project_net(prior_path: Path) -> Dict[str, float]:
+    """Extract {project: net hours} from a prior preferred-format workbook."""
+    import openpyxl
+    out: Dict[str, float] = {}
+    try:
+        wb = openpyxl.load_workbook(str(prior_path), data_only=True, read_only=True)
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)